coreax/approximation.py

```diff
@@ -169,19 +169,18 @@ class MonteCarloApproximateKernel(RandomRegressionKernel):
     :param num_train_points: Number of training points used to fit kernel regression
     """
 
-    def calculate_kernel_matrix_row_sum_mean(
-        self, x: ArrayLike, max_size: int = 10_000
-    ) -> Array:
+    def gramian_row_mean(self, x: ArrayLike, **kwargs) -> Array:
         r"""
-        Calculate the kernel matrix row sum mean by Monte Carlo approximation.
+        Approximate the Gramian row-mean by Monte-Carlo sampling.
 
-        A uniform random subset of ``x`` is used to approximate the kernel matrix row
-        sum mean.
+        A uniform random subset of ``x`` is used to approximate the base kernel's
+        Gramian row-mean.
 
         :param x: Data matrix, :math:`n \times d`
         :param max_size: Size of matrix block to process
```

Comment:
Should be block_size?

Reply:
Yes, I'll double check they're aren't any more of these! Thanks for spotting!